tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/ComplianceRequirement.scala

```diff
@@ -33,7 +33,7 @@ object ComplianceRequirement {
    *  requirements.
    *  @throws BadComplianceException if the semantics are not compliant.
    */
-  final def checkCompliance(requirements: Traversable[ComplianceRequirement],
+  final def checkCompliance(requirements: Iterable[ComplianceRequirement],
```

Comment:
That is not an acceptable change, because it breaks a public API.

You need to use a `type` alias in an `Compat` object instead.